src/PAModel/Serializers/SourceSerializer.cs

```diff
@@ -778,13 +778,6 @@ private static void LoadDataSources(CanvasDocument app, DirectoryReader director
                         dataSourceReferences.Add(tableDef.DatasetName, localDatabaseReferenceJson);
                     }
                 }
-                if (localDatabaseReferenceJson.instanceUrl != tableDef.InstanceUrl)
-                {
-                    // Generate an error, dataset defs have diverged in a way that shouldn't be possible
-                    // Each dataset has one instanceurl
-                    errors.ValidationError($"For file {file._relativeName}, the dataset {tableDef.DatasetName} has multiple instanceurls");
-                    throw new DocumentException();
```

Comment:
Can you add a test that fails for this scenario prior to the change for each of these blocks that you removed? The "throw" removals are the places where I'd be most concerned about removing. We don't want it to fail later with a "null ref" or something like that for these cases...